products/extensions/b2b-components/shopping-lists/02-api-and-pricing.md

```diff
@@ -0,0 +1,120 @@
+---
+nav:
+  title: API & Pricing
+  position: 30
+
+---
+
+# Store API
```

Comment:
Should we just create a link to our store API docs? 
https://shopware.stoplight.io/docs/store-api/c9849725606fd-create-new-shopping-list